src/lsptoolshost/roslynLanguageServer.ts

```diff
@@ -672,6 +685,30 @@ export async function activateRoslynLanguageServer(
 
     // Start the language server.
     _languageServer.start();
+
+    return _languageServer;
+
+    function scanExtensionPlugins(): string[] {
```

Comment:
I know we had some concerns about performance, how expensive was this in the end?

Reply:
Ended up not contributing noticeably in my (relatively unscientific) benchmarks.